### PR TITLE
Anqi - editPerformance

### PR DESCRIPTION
**Pull request by Anqi-nus**

### CODE REVIEW

File: src/main/java/seedu/event/EventList.java [hunk 1]
```diff
@@ -185,7 +185,9 @@ public Event getEvent(String eventName) throws PacException {
             throw new PacException("The event list is empty.");
         }
         for (Event event : list) {
-            if (event.getName().equals(eventName)) {
```
Comment (benchan911) [inline src/main/java/seedu/event/EventList.java:188]: @harithadiv @lowjiayee need your review here to make sure it doesn't affect the rest of the code

File: src/main/java/seedu/performance/PerformanceList.java [hunk 2]
```diff
@@ -21,8 +21,27 @@ public PerformanceList() {
     }
 
     public void addToList(Performance performance, String eventName) {
-        performanceList.add(performance);
-        ui.addPerformanceMessage(performance.studentName, eventName);
+        if (isRepeated(performance)) {
```
Comment (benchan911) [inline src/main/java/seedu/performance/PerformanceList.java:24]: Would it be better to write isEmpty(performance), so you edit when it is not empty not when the process is repeated

File: src/main/java/seedu/ui/UI.java [hunk 3]
```diff
@@ -118,28 +118,19 @@ public void deleteEventMessage(String eventType, String eventName) {
                 + "your Event list.\n", eventType, eventName);
     }
 
-    public void printInvalidDateTimeFormat() {
```
Comment (benchan911) [inline src/main/java/seedu/ui/UI.java]: why are you deleting this? Just need to verify here

**Review by benchan911 — COMMENTED** [review]
Please take a look at my comments, it is mostly on naming. I personally feel the general idea with 'edit' function is that you cannot edit an empty list so the edit function will always work if the list is not empty.

File: src/main/java/seedu/ui/UI.java (hunk 3 above)
Comment (Anqi-nus) [reply src/main/java/seedu/ui/UI.java]: this is from merge conflict.

File: src/main/java/seedu/performance/PerformanceList.java (hunk 2 above)
Comment (Anqi-nus) [reply src/main/java/seedu/performance/PerformanceList.java:24]: this is addToList

**Review by benchan911 — APPROVED** [review]
Can merge

**Pull request merged**